Week06/pi_burak_gulluler.py

```diff
@@ -0,0 +1,20 @@
+import random
+import math
+
+def next_pi():
+    x = random.random()
```

Comment:
Why do we need these 3 lines?

Reply:
Before I learned that we need to make a function to solve the problem, I had made a class then convert it to the function that's why they are here :) I'll handle it immediately